common/src/main/webapp/less/style.less

```diff
@@ -998,7 +999,7 @@ tfoot {
 }
 
 .aladin-detail-text {
-  color: white;
+  color: hsl(0, 0%, 100%);
```

Comment:
I'm guessing this is unrelated?

Reply:
This was to Truly test the fixing part.I'll revert it